bcda/cclf/cclf.go

```diff
@@ -451,6 +459,12 @@ func sortCCLFFiles(cclfmap *map[string][]*cclfFileMetadata, skipped *int) filepa
 		metadata.filePath = path
 		metadata.deliveryDate = info.ModTime()
 		if err != nil {
+			if isSuppressionFile(info.Name()) {
```

Comment:
Instead of checking for suppression file, lets just keep all files around if they are less than 24 hours.  On next execution (within 24 hours on cron), the file will be moved to pending deletion dir because we don't want to keep that sensitive data around for longer than 24 hours.